lib-es5/api_client/execute_request.js

```diff
@@ -1,5 +1,7 @@
 'use strict';
 
+var _extends = Object.assign || function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; };
```

Comment:
Is that a result of `npm run compile`?

Reply:
It is indeed

Reply:
Cool, resolving the conversation then ;)